hack/pre-release.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+set -euo pipefail
+# this script updates the WMCO and OCP versions in the Makefile, base CSV, bundle and Dockerfiles.
+
+# usage:
+# on a release-4.x branch, run ./pre-release (x, y or z)
+# to update a z stream, use ./pre-release.sh z
+# for other streams, use the corresponding x y or z argument.
+
+# Include the common.sh 
+  WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+  source $WMCO_ROOT/hack/common.sh
+
+update_WMCO_version(){
+  xyz_stream=$1
+  cluster_service_version_yaml=config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml
+ 
+  # Setting the stream var to 1 2 or 3 which is used for incrementing the correct stream
+  # this tells awk which column to act on. Column 1, column 2, or column 3.
```

Comment:
This can be made clearer. The columns mean what? I assume major, minor, and patch version